components/molecules/SharePopup/index.tsx

```diff
@@ -39,19 +50,50 @@ const SharePopup: FC<SharePopupProps> = ({ title, url, className }) => {
     }
   );
 
+  const finalUrl = createFinalUrl(url || pathname || undefined);
+
+  const handleClick: MouseEventHandler = async (event) => {
```

Comment:
click handler will block the default event, to prevent the popover from showing, if the device's user agent matches for mobile and has the sharing API available